packages/@sanity/base/src/components/previews/detailPreview.tsx

```diff
@@ -58,9 +58,11 @@ export const DetailPreview: React.FunctionComponent<PreviewProps<'detail'>> = (p
             })}
           {typeof media === 'string' && <MediaString>{media}</MediaString>}
           {React.isValidElement(media) && media}
+          <span />
```

Comment:
This is to render the shadow right? Could you use a pseudo element?

Reply:
Yes, it’s to render the shadow.

The thing is we only want the shadow when then `<MediaWrapper>` contains something else `<svg>` – icons should not have the shadow.

So the idea is the use the `*:not(svg) + span` selector target only that situation to render the shadow.

Will add a comment about this in the code 👍